### PR TITLE
Make sure file exists before using it

### PR DESCRIPTION
**Pull request by mnowaksuse**
poo#18134

Tested manually.

### CODE REVIEW

File: data/textinfo [hunk 1]
```diff
@@ -13,9 +13,9 @@ ip -6 r s
 cat /etc/resolv.conf
 ls -al /etc/sysconfig/network
 rpm -qa 'kernel-*'
-grep DISPLAYMANAGER /etc/sysconfig/displaymanager
-grep DEFAULT /etc/sysconfig/windowmanager
-ls -l /etc/ntp*
```
Comment (lnussel) [inline data/textinfo]: I'd rather user grep ... || true, the && would also make the script exit prematurely

File: data/textinfo [hunk 2]
```diff
@@ -13,10 +13,10 @@ ip -6 r s
...
-[ -f /var/log/messages ] && du /var/log/messages
+grep DISPLAYMANAGER /etc/sysconfig/displaymanager || true
+grep DEFAULT /etc/sysconfig/windowmanager || true
+ls -l /etc/ntp* || true
+du /var/log/messages || true
```
Comment (okurz) [inline data/textinfo]: why?

Comment (mnowaksuse) [inline data/textinfo]: Can you be more specific?

Comment (okurz) [inline data/textinfo]: why do you replace my check for the file existance with just calling du on a potentially non-existing file and then ignoring the exit status?

Comment (mnowaksuse) [inline data/textinfo]: ok

**Comment by okurz** [comment]
thank you

**Pull request merged**